Tests/Unit/DependencyInjection/CmfSeoExtensionTest.php

```diff
@@ -121,11 +121,13 @@ public function testAlternateLocaleWithPhpcr()
             'Symfony\Cmf\Bundle\SeoBundle\Doctrine\Phpcr\AlternateLocaleProvider'
         );
 
-        $this->assertContainerBuilderHasServiceDefinitionWithMethodCall(
-            'cmf_seo.event_listener.seo_content',
-            'setAlternateLocaleProvider',
-            array($this->container->getDefinition('cmf_seo.alternate_locale.provider_phpcr'))
-        );
+        if ($this->container->has('cmf_seo.event_listener.seo_content')) {
```

Comment:
Because this PR adds the ability to remove the seo content listener, therefore this test needs to only assert that the event listener exists with this method call if the event listener exists